src/main/java/com/internet/shop/util/ConnectionUtil.java

```diff
@@ -0,0 +1,29 @@
+package com.internet.shop.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    static {
+        try {
+            Class.forName("com.mysql.jdbc.Driver");
+        } catch (ClassNotFoundException e) {
+            throw new RuntimeException("Can't find MySql Driver: ", e);
+        }
+    }
+
+    public static Connection getConnection() {
+        Properties dbProperties = new Properties();
+        dbProperties.put("user", "root");
+        dbProperties.put("password", "1234");
+
+        String url = "jdbc:mysql://localhost:3306/internet_shop?serverTimezone=UTC";
```

Comment:
If you create a new variable url, maybe you should add it to constants?

Reply:
Agree